sys/usb/usbus/cdc/acm/cdc_acm_stdio.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2018 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup sys
+ * @{
+ *
+ * @file
+ * @brief CDC ACM stdio implementation
+ *
+ * This file implements a USB CDC ACM callback and read/write functions.
+ *
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#if MODULE_VFS
+#include <unistd.h>
+#include <errno.h>
+#include <fcntl.h>
+#endif
+#include "stdio_uart.h"
+
+#include "board.h"
+#include "isrpipe.h"
+
+#include "usb/usbus.h"
+#include "usb/usbus/cdc/acm.h"
+
+#if MODULE_VFS
+#include "vfs.h"
+#endif
+
+static usbus_cdcacm_device_t cdcacm;
+static uint8_t _cdc_tx_buf_mem[STDIO_UART_RX_BUFSIZE * 4];
+static uint8_t _cdc_rx_buf_mem[STDIO_UART_RX_BUFSIZE];
+isrpipe_t cdc_stdio_isrpipe = ISRPIPE_INIT(_cdc_rx_buf_mem);
+
+#if MODULE_VFS
+static ssize_t uart_stdio_vfs_read(vfs_file_t *filp, void *dest, size_t nbytes);
+static ssize_t uart_stdio_vfs_write(vfs_file_t *filp, const void *src, size_t nbytes);
```

Comment:
`s/uart_//g`?

Reply:
Reworked to match the way it is implemented in `stdio_uart` (completely removed this bit)